pages/onboarding/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Button, Container, Typography } from '@mui/material';
+import { NextPage } from 'next';
+import Link from 'next/link';
+
+const Index: NextPage = () => {
+  return (
+    <Container maxWidth="lg" className="flex justify-between pt-4">
+      <Typography variant="h1" className="text-3xl">
+        Onboarding
+      </Typography>
+      <Box className="flex items-center gap-4">
+        <Link href="/" passHref>
```

Comment:
what's the purpose of the `passHref` prop here? From the docs it says:

> Forces Link to send the href property to its child. Defaults to false

But a button shouldn't need an href?

Reply:
It shouldn't really be a button to begin with, it's linking to another page... it's not (currently) doing any background tasks on the current page. MUI's button accepts a `href` ([docs](https://mui.com/material-ui/api/button/#props)).
> href | string | The URL to link to when the button is clicked. If defined, an a element will be used as the root node.

This is basically making it a link while keeping it styled like a button.

Reply:
ah, gotcha. thanks for explaining!

Reply:
So I tested with and without, In both cases the page transfer still activates but the HTML is rendered differently

This is without `passHref`

```html
<div class="flex items-center gap-4 MuiBox-root css-0">
<button class="(...many classes)" tabindex="0" type="button">
Skip
<span class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"></span>
</button>
</div>
```

This is with `passHref`

```html
<div class="flex items-center gap-4 MuiBox-root css-0">
<a class="(...many classes)" tabindex="0" href="/">
Skip
<span class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"></span>
</a>
</div>
```

Reply:
Correct, `passHref` on the NextJS's `Link` component is passing `href` into its children; which, in this case, is the MUI `Button`.

The MUI button will then change the "root node" to an `a` instead of a `button` element when `href` is passed into it. Thus resulting in the changes you observed.